packages/public/server/src/module/Uuid/config/module.config.php

```diff
@@ -105,6 +105,9 @@
                     'setUuidManager' => [
                         'required' => true,
                     ],
+                    'setInstanceManager' => [
+                        'required' => true,
+                    ]
```

Comment:
@vroland Run `yarn format:php` after you have finished the work on this PR to format the PHP code and to remove the lint error.